### DISCUSSION TITLE
Modoboa v2 GUI functional or only a visual preview?

### DISCUSSION THREAD

**Discussion by JohnHammell** [modoboa/modoboa#2513] [Help] [answered]
When I go to add a domain in Modoboa v2 and click "Confirm and Create" on the Summary page, nothing happens. The button flashes / changes color for a moment, but nothing after that. No indication that the domain creation was communicated to the server.

Wondering if there's an install issue that I need to investigate or if maybe the Modoboa v2 GUI is not yet functional but only offered right now as a visual preview of the GUI.

Is Modoboa v2 supposed to be working / ready to be used?

I can confirm, though, that domains & addresses/accounts created in the original GUI can be edited in Modoboa v2.

Thanks

**Comment by tonioo** [comment, marked as answer]
There is a bug regarding domain creation in the new UI (https://github.com/modoboa/modoboa/issues/2507). 
But otherwise, it is supposed to be functional :)
